collections/_article/godot-xr-community-nov-2024.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Godot XR community - November 2024"
+excerpt: "Interesting projects from the Godot XR community."
+categories: ["news"]
+author: Bastiaan Olij
+image: /storage/blog/covers/november-2024-xr-community-update.webp
+image_caption_description: "tbd"
+date: 2024-11-01 12:00:00
+---
+
+Godot has a very active XR-focused community that mainly help each other out through the XR channel of the [official Godot Discord server](/community/).
+Today, we want to put a spotlight on a number of community projects.
+
+## Godot XR community Game Jams
+
+To showcase the engine capabilities and the ingenuity of the Godot XR community, game jams are now being organized twice a year.
+For participants, it is a fun way to get introduced to Godot's XR capabilities.
+
+### Godot XR Game Jam #1
+
+The [first game jam](https://itch.io/jam/godot-xr-game-jam) was held from December 1st until December 4th 2023.
+The theme of this game jam was Vapor.
+We saw both many new faces and long time community members join this first jam, with a total of 17 entries.
+
+The #1 spot went to [Cloud Shepherd](https://itch.io/jam/godot-xr-game-jam/rate/2402526) which won 3 out of 6 categories.
+This entry let you herd sheep on clouds in the sky. With a unique movement mechanic and using a fan to create
+a breeze with which to move the sheep around, this was a fascinating entry.
+
+The #2 spot went to [Super Steam Hands](https://itch.io/jam/godot-xr-game-jam/rate/2402404) which won 2 out of 6 categories.
+Don't let the thumbnail of this entry fool you. A great game where you can feel like Iron Man by using jets of steam to parkour around a level.
+
+The #3 spot went to [Pipe Stream](https://itch.io/jam/godot-xr-game-jam/rate/2400974) which won 1 out of 6 categories.
+A great little puzzle game where you need to chip away a path for vapour to reach a target. Very challenging!
```

Comment:
```suggestion
It's a great little puzzle game where you need to chip away a path for vapour to reach a target. Very challenging!
```

Reply:
I notice it says "vapour" in this sentence, but "Vapor" above. I don't know which we should use but probably should be consistent.

Reply:
Seeing the announced theme was "Vapor", we should probably stick with that.